test/unit/model/result/ScoreWritterTest.php

```diff
@@ -0,0 +1,120 @@
+<?php
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA;
+ */
+
+namespace oat\taoLtiConsumer\test\unit\model\result;
+
+use common_exception_Error;
+use common_exception_NotFound;
+use core_kernel_classes_Resource;
+use oat\generis\test\MockObject;
+use oat\generis\test\TestCase;
+use oat\taoDelivery\model\execution\DeliveryExecutionInterface;
+use oat\taoLtiConsumer\model\result\ScoreWriterService;
+use oat\taoResultServer\models\classes\ResultServerService;
+use oat\taoResultServer\models\Exceptions\DuplicateVariableException;
+use taoResultServer_models_classes_OutcomeVariable;
+use taoResultServer_models_classes_Variable;
+use taoResultServer_models_classes_WritableResultStorage;
+
+class ScoreWritterTest extends TestCase
```

Comment:
i think we have a typo inside the name of the object maybe it should be 'ScoreWriter'

Reply:
fixed